src/addrman.h

```diff
@@ -146,7 +146,9 @@ class AddrMan
     /**
      * Choose an address to connect to.
      *
-     * @param[in] new_only Whether to only select addresses from the new table.
+     * @param[in] new_only Whether to only select addresses from the new table. Passing `true` returns
+     *                     an address from the new table or an empty pair. Passing `false` will return an
+     *                     address from either the new or tried table (it does not guarantee a tried entry).
```

Comment:
nit: mentioning the "empty pair" only in the first case makes it sound as if this wasn't one of the possible outcomes in the second case (but it is).